indent/javascript.vim

```diff
@@ -59,6 +59,14 @@ let s:syng_linecom = 'linecomment\c'
 " Expression used to check whether we should skip a match with searchpair().
 let s:skip_expr = "synIDattr(synID(line('.'),col('.'),1),'name') =~ '".s:syng_strcom."'"
 
+func s:searchpar(start,mid,end,flags,stop)
```

Comment:
This name is confusingly similar to `searchpair`. Perhaps `s:searchparens` would be clearer.